src/style.css

```diff
@@ -1,3 +1,85 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  margin: 0;
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  padding: 0 50px;
+  box-sizing: border-box;
+  width: 100%;
+  align-items: center;
+}
+
+.logo {
+  display: block;
+  height: 40px;
+  width: 40px;
+}
+
+.nav_item {
+  list-style-type: none;
+  display: flex;
+  align-items: center;
+  margin: 0 0 0 20px;
```

Comment:
You are adding vertical margins to your `.nav_item` class. Remember to be consistent with your vertical margins. Add only top or only bottom margin, but not both. Vertical margins can be collapsed in some cases, causing unexpected results.